src/sentry/features/__init__.py

```diff
@@ -178,16 +181,16 @@
 default_manager.add("organizations:integrations-chat-unfurl", OrganizationFeature)
 default_manager.add("organizations:integrations-codeowners", OrganizationFeature)
 default_manager.add("organizations:integrations-custom-scm", OrganizationFeature)
-default_manager.add("organizations:integrations-feature-flag-integration", OrganizationFeature)
 default_manager.add("organizations:integrations-event-hooks", OrganizationFeature)
+default_manager.add("organizations:integrations-feature-flag-integration", OrganizationFeature)
 default_manager.add("organizations:integrations-incident-management", OrganizationFeature)
 default_manager.add("organizations:integrations-issue-basic", OrganizationFeature)
 default_manager.add("organizations:integrations-issue-sync", OrganizationFeature)
 default_manager.add("organizations:integrations-stacktrace-link", OrganizationFeature)
 default_manager.add("organizations:integrations-ticket-rules", OrganizationFeature)
 default_manager.add("organizations:performance-issue-details-backend", OrganizationFeature)
```

Comment:
Can you scooch this up out of the "NOTE: Don't add features down here!" section while you're at it?

Reply:
where should it go? just up a single line?

Reply:
ohhh I see

Reply:
I'll do in a diff PR actually